pkg/snet/arclient/client.go

```diff
@@ -1,4 +1,5 @@
 // Package arclient implements address resolver client
+// TODO(nkryuchkov): move to internal
```

Comment:
do you plan to move it in a separate PR?

Reply:
@Darkren Yes, I'm going to do that in a PR for https://github.com/SkycoinProject/skywire-mainnet/issues/409